setup.py

```diff
@@ -29,7 +29,7 @@
         "boto3>=1.7.0",
         "botocore>=1.12.0",
         "dnspython>=1.15.0",
-        "neo4j-driver>=1.7.0",
+        "neo4j>=1.7.0",
```

Comment:
After digging into this it looks like the `neo4j-driver` package will cease being published, but the next version of the `neo4j` package is 4.0. While we're at it, can you please add a `<4.0` constraint here?